docs/understanding-tracking-design/versioning-your-data-structures/superseding-schema/index.md

```diff
@@ -104,27 +112,6 @@ When this schema submitted to Iglu Server, information of `1-0-1` superseding `1
 </details>
 Note the `$supersededBy` field in there. This field specifies that if an event is received with `1-0-0` version of that schema, version should be replaced with `1-0-1` and the respective entity will be validated with `1-0-1` instead. This operation is performed on Enrich.
 
-## Upgrading components for superseding schema feature
-
-You need >= Enrich 3.8.0 and >= Iglu Server 0.10.0 to be able to use this feature.
-
-While upgrading Iglu Server to 0.10.0, you need to add new `superseded_by` column to `iglu_schemas` table. You can do this either with the following command:
```

Comment:
We decided to make the migration automatically when Iglu Server started. Therefore, upgrade guide isn't needed anymore.